### DISCUSSION TITLE
error on using postcss in tailwindcss

### DISCUSSION THREAD

**Discussion by hummadarshad** [tailwindlabs/tailwindcss#11049] [Help]
I am using PostCSS method  in Tailwindcss  I have a issue , If I have a single index.html file its works for production but if I have multiple html files like conatc.html then in build process vite is only building index file and not building other files so what I have to do to add my ither html files . Can anyone help me ?

**Comment by wongjn** [comment]
[See the Vite documentation for multi-page apps](https://vitejs.dev/guide/build.html#multi-page-app). If you require more assistance regarding this, consider asking in discussion area more geared towards Vite.

**Reply by hummadarshad** [reply]
Thanks A Lot

**Comment by asmin52** [comment]
could you share more information.
[Dashboard Anywhere Login](https://www.dashboard-anywhere.org/)
